wgengine/router/router_linux.go

```diff
@@ -200,8 +200,8 @@ type linuxRouter struct {
 	// ipPolicyPrefBase is the base priority at which ip rules are installed.
 	ipPolicyPrefBase int
 
-	nfr netfilterRunner
 	cmd commandRunner
+	nfr netfilterRunner
```

Comment:
Why the move? I don't particularly care, just curious.

Reply:
Oh, this went through a few changes, I was originally going to make it atomic / locked, but undid all that as I need to spend far more time scrutinizing our surrounding lock strategy. Right now the new behavior will at least remain consistent with the reconfig path.